snuba/datasets/configuration/entity_builder.py

```diff
@@ -30,10 +26,7 @@
 from snuba.query.processors.quota_processor import ResourceQuotaProcessor
 from snuba.query.processors.tags_type_transformer import TagsTypeTransformer
 from snuba.query.processors.timeseries_processor import TimeSeriesProcessor
-from snuba.query.validation.validators import (
-    EntityRequiredColumnValidator,
-    QueryValidator,
```

Comment:
If you're not auto-importing, how are these getting registered?

I'm wondering if this is only succeeding because other entities are loading the validators. What happens if you run `snuba entities load snuba/datasets/configuration/generic_metrics/entities/sets.yaml`?

Reply:
Also, does `pytest tests/datasets/configuration/test_entity_loader.py` on its own work? That loads the python-created entity *after* using the entity builder, but if you use the entire suite of tests then the other classes might get loaded and make these imports unnecessary.

Reply:
never mind, I cloned your branch locally and I now see that the validators/mappers we're using are already part of the base import path, which is probably why you punted on QueryProcessors.